Source/WebCore/loader/ResourceMonitor.h

```diff
@@ -38,6 +38,13 @@ class LocalFrame;
 class ResourceMonitor final : public RefCountedAndCanMakeWeakPtr<ResourceMonitor> {
 public:
     using Eligibility = ResourceMonitorEligibility;
+    enum class UsageLevel : uint8_t {
+        Empty = 0,
+        Low = 20,
+        Medium = 60,
+        High = 80,
```

Comment:
I am worried this is too much logging and I'm unclear how it will help us in sysdiagnoses. Why do we need 5 levels when this is basically a binary thing (too much data or not?)

I think we basically want logging when a frame gets unloaded due to large usage, that's it.

Reply:
I want to understand the actual usage in the real world web site. I need these value to understand how usage grows without logging whole usage. It creates max of 5 more logs for each frame so I don't think this causes too much logging. And also after understanding the usage characteristics at the end, we can hide this under compiler flag. I want to ask you to let this in the code.